packages/test-utils/scss.js

```diff
@@ -8,18 +8,32 @@
 'use strict';
 
 const fs = require('fs');
-const { render, types } = require('node-sass');
+const { render: nodeRender, types: nodeTypes } = require('node-sass');
+const { render: dartRender, types: dartTypes } = require('sass');
+// TODO figure out how to use fibers with jest
+// const Fiber = require('fibers');
```

Comment:
Did you figure this out with the module extensions stuff that you left a note for above?

Reply:
I haven't unfortunately ... I _think_ it's related to the babel generator transforms, but other parts of jest rely on regenerator-runtime so I don't think I can remove it ... that said, build times don't seem to be an issue, so we should be fine without fibers (for now)